zulip_bots/zulip_bots/run.py

```diff
@@ -3,12 +3,33 @@
 from __future__ import absolute_import
 
 import logging
-import optparse
+import argparse
 import sys
+import os
 from types import ModuleType
 from importlib import import_module
+from os.path import basename, splitext
+
+import six
 
 from zulip_bots.lib import run_message_handler_for_bot
+from zulip_bots.provision import provision_bot
+
+
+def import_module_from_source(path, name=None):
+    if name is None:
+        name = splitext(basename(path))[0]
+
+    if six.PY2:
```

Comment:
Did we make a firm decision yet to go to python3 only for bots?  If so, we can eliminate the PY2 stuff.